LocationUpdatesBackgroundKotlin/app/src/main/java/com/google/android/gms/location/sample/locationupdatesbackgroundkotlin/Utils.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.gms.location.sample.locationupdatesbackgroundkotlin
+
+import android.Manifest
+import android.content.Context
+import android.content.pm.PackageManager
+
+import androidx.core.app.ActivityCompat
+import androidx.fragment.app.Fragment
+
+import com.google.android.material.snackbar.Snackbar
+
+/**
+ * Helper functions to simplify permission checks/requests.
+ */
+fun Context.hasPermission(permission: String): Boolean {
+
+    // Background permissions didn't exit prior to Q, so it's approved by default.
+    if (permission == Manifest.permission.ACCESS_BACKGROUND_LOCATION &&
+        android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.Q) {
+
```

Comment:
new line not needed

Reply:
ran ktlint and fixed a bunch of other stuff, but it says this is ok... ? 

No idea, but I removed anyway. :)